cilium/CFP-31752-active-connection-tracking.md

```diff
@@ -0,0 +1,200 @@
+# CFP-31752: Active Connection Tracking
+
+**SIG: SIG-DATAPATH**
+
+**Begin Design Discussion:** 2024-03-13
+
+**Cilium Release:** 1.16
+
+**Authors:** Aleksander Mistewicz <amistewicz@google.com>, Rob Scott <robertjscott@google.com>
```

Comment:
```suggestion
**Authors:** Aleksander Mistewicz <amistewicz@google.com>, Rob Scott <robertjscott@google.com>

**Status:** Released Cilium 1.16
```

Reply:
Only the first two PR are part of v1.16 release. The other two - surfacing metrics in prometheus and counting connection errors - are only on main. Do you want to backport them to v1.16 or should it say `Released Cilium 1.17`?

Reply:
You're right. Updated to 1.17